build.sbt

```diff
@@ -27,19 +26,16 @@ lazy val microservice: Project = Project(appName, file("."))
     defaultSettings(),
     scoverageSettings,
     scalacOptions += "-Ywarn-unused:-explicits,-implicits",
-    scalaVersion := "2.12.15",
+    scalaVersion := "2.13.8",
     libraryDependencies ++= appDependencies,
     retrieveManaged := true,
     majorVersion := 4,
-    Compile / scalacOptions += "-P:silencer:pathFilters=target/.*"
+    Compile / scalacOptions += "-Wconf:src=target/.*:s"
   )
   .settings(
-    // Use the silencer plugin to suppress warnings from unused imports in compiled twirl templates
-    scalacOptions += "-P:silencer:pathFilters=views;routes",
-    libraryDependencies ++= Seq(
-      compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
-      "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full
-    ),
+    scalacOptions += "-Wconf:src=routes/.*:s",
```

Comment:
I'm guessing that these could be all hooked up together to avoid repetition of scalacOptions
anywhere under settings eg
scalacOptions += Seq("", "", "")